VideoLocker/res/values/styles.xml

```diff
@@ -404,8 +404,6 @@
         <item name="itemVerticalPadding">@dimen/popupMenuItemVerticalPadding</item>
         <item name="iconPadding">@dimen/popupMenuIconPadding</item>
         <item name="iconDefaultSize">@dimen/popupMenuIconDefaultSize</item>
-        <item name="headerTextAppearance">@style/edX.Widget.PopupMenu.HeaderTextAppearance</item>
-        <item name="rowTextAppearance">@style/edX.Widget.PopupMenu.RowTextAppearance</item>
```

Comment:
I guess these attributes can be removed from the `PopupMenu` attribute definitions in `attrs.xml` now.